src/implementations/vanilla/components/tabs/tabs.js

```diff
@@ -334,6 +374,100 @@ export class Tabs {
     this.moreButton.setAttribute('aria-expanded', false);
     this.dropdown.classList.remove('ecl-tabs__dropdown--show');
   }
+
+  /**
+   * Tabs onKeydown handler.
+   * @param {Event} e
+   */
+  onKeydown(e) {
+    const tgt = e.currentTarget;
+    let flag = false;
+
+    switch (e.key) {
+      case 'ArrowLeft':
+        this.moveFocusToPreviousTab(tgt);
+        flag = true;
+        break;
+
+      case 'ArrowRight':
+        this.moveFocusToNextTab(tgt);
+        flag = true;
+        break;
+
+      case 'Home':
+        this.moveFocusToTab(this.firstTab);
+        flag = true;
+        break;
+
+      case 'End':
+        this.moveFocusToTab(this.lastTab);
+        flag = true;
+        break;
+
+      default:
+        break;
+    }
+
+    if (flag) {
+      e.stopPropagation();
```

Comment:
not sure about what we would prevent here, but ok, just that for how the logic is it seems you could set the variable to true and then in the default case set it to false, not a big difference but less repetitions..

Reply:
Sure you are right, I have removed this prevents condition 👍